src/calendar/SelectedMonthView.java

```diff
@@ -0,0 +1,284 @@
+				daysHolder[week][counter].setOpaque(true);
+				daysHolder[week][counter].setBorderPainted(false);
+				daysHolder[week][counter].setForeground(Color.white);
+				counter++;
+				i++;
+			} else if (counter < 6 && i >= 10 && i < lengthOfMonth && i != today) {
+				daysHolder[week][counter].setText("" + i);
+				counter++;
+				i++;
+			} else if (counter < 6 && i >= 10 && i < lengthOfMonth && i == today && !sameMonth) {
+				daysHolder[week][counter].setText("" + i);
+				counter++;
+				i++;
+			} else if (counter < 6 && i >= 10 && i < lengthOfMonth && i == today && sameMonth && !sameYear) {
+				daysHolder[week][counter].setText("" + i);
+				counter++;
+				i++;
+			} else if (counter < 6 && i >= 10 && i < lengthOfMonth && i == today && sameMonth && sameYear) {
+				daysHolder[week][counter].setText("" + i);
+				daysHolder[week][counter].setBackground(Color.blue);
+				daysHolder[week][counter].setOpaque(true);
+				daysHolder[week][counter].setBorderPainted(false);
+				daysHolder[week][counter].setForeground(Color.white);
+				counter++;
+				i++;
+			} else if (counter == 6 && i >= 10 && i < lengthOfMonth && i != today) {
+				daysHolder[week][counter].setText("" + i);
+				counter = 0;
+				i++;
+				week++;
+			} else if (counter == 6 && i >= 10 && i < lengthOfMonth && i == today && !sameMonth) {
+				daysHolder[week][counter].setText("" + i);
+				counter = 0;
+				i++;
+				week++;
+			} else if (counter == 6 && i >= 10 && i < lengthOfMonth && i == today && sameMonth && !sameYear) {
+				daysHolder[week][counter].setText("" + i);
+				counter = 0;
+				i++;
+				week++;
+			} else if (counter == 6 && i >= 10 && i < lengthOfMonth && i == today && sameMonth && sameYear) {
+				daysHolder[week][counter].setText("" + i);
+				daysHolder[week][counter].setBackground(Color.blue);
+				daysHolder[week][counter].setOpaque(true);
+				daysHolder[week][counter].setBorderPainted(false);
+				daysHolder[week][counter].setForeground(Color.white);
+				counter++;
+				i++;
+			} else if (counter <= 6 && i >= 10 && i == lengthOfMonth && i != today) {
+				daysHolder[week][counter].setText("" + i);
+				break;
+			} else if (counter == 6 && i >= 10 && i == lengthOfMonth && i == today && !sameMonth) {
+				daysHolder[week][counter].setText("" + i);
+				break;
+			} else if (counter == 6 && i >= 10 && i == lengthOfMonth && i == today && sameMonth && !sameYear) {
+				daysHolder[week][counter].setText("" + i);
+				break;
+			} else if (counter == 6 && i >= 10 && i == lengthOfMonth && i == today && sameMonth && sameYear) {
+				daysHolder[week][counter].setText("" + i);
+				daysHolder[week][counter].setBackground(Color.blue);
+				daysHolder[week][counter].setOpaque(true);
+				daysHolder[week][counter].setBorderPainted(false);
+				daysHolder[week][counter].setForeground(Color.white);
+				counter++;
+				i++;
+				break;
+			} else {
+				System.out.println("error");
+				break;
+			}
+		}
+	}
+
+}
```

Comment:
don't forget to make sure that when a button with a date is clicked, it changes the selectedDate with CalendarModel.setSelectedDate()